source/transfer_list.rst

```diff
@@ -398,6 +398,19 @@ Inputs:
 
    #. `void_te.size` =  `old_void_data_size - align8(new_data_size + 0x8)`
 
+Removing a TE
```

Comment:
Seems to be missing a commit message

Reply:
Have only got commit header but no body, hope that is fine.

Reply:
Please add a body to all commits.